tests/cases/conformance/classes/members/privateNames/privateNameInInExpressionTransform.ts

```diff
@@ -0,0 +1,47 @@
+// @target: es2020

```

Comment:
```suggestion
// @target: esnext, es2020
```
This will generate outputs for multiple targets so we can see the same transformation side by side

Reply:
Ah great, TIL. I will make that change.